README.md

```diff
@@ -16,7 +16,7 @@ As a Research Support Facility, we will be covering multiple research and commun
 This theme will bring different research collaborations across the UK together within a trusted research environment to facilitate data, software and analysis sharing. This will support a progressive move from reproducible, to reproduced, to reused research artefacts and outputs, maximizing the return on the NIHR’s investment in the overall research programme.
 
 ### Theme 2: Accessible, research ready data
-##### Team: Dr Ann-Marie Mallon, Mahwish Mohammad, Dr Rachael Stickland
+##### Team: Dr Ann-Marie Mallon, ([amallon@turing.ac.uk](mailto:amallon@turing.ac.uk)), Mahwish Mohammad ([mmohammad@turing.ac.uk](mailto:mmohammad@turing.ac.uk)), Dr Rachael Stickland ((rstickland@turing.ac.uk)[mailto:rstickland@turing.ac.uk])
```

Comment:
@eirini-zormpa - looks like you have the brackets the wrong way round,it should be - 
([rstickland@turing.ac.uk](mailto:rstickland@turing.ac.uk))

Reply:
hahahaha of course 🤦🏻‍♀️

PS when adding a comment on a line the first icon after preview (a file with a plus and minus sign on it) allows you to add suggestions instead of adding them in the comment! I recently found out about it so I thought I'd mention in case you didn't know :blush: